localstack-core/localstack/services/apigateway/next_gen/execute_api/context.py

```diff
@@ -51,6 +51,29 @@ class IntegrationRequest(TypedDict, total=False):
     """Body content of the request"""
 
 
+class BaseResponse(TypedDict):
+    """Base class for Response objects in the context"""
+
+    status_code: int
+    """Status code of the response"""
+    headers: Headers
+    """Headers of the response"""
+    body: bytes
+    """Body content of the response"""
+
+
+class EndpointResponse(BaseResponse):
+    """Represents the response coming from an integration, called Endpoint Response in AWS"""
+
+    pass
+
+
+class InvocationResponse(BaseResponse):
+    """Represents the response coming after being serialized in an integration response in AWS"""
```

Comment:
Just keeping the syntax style similar :wink: 
```suggestion
    """Represents the response coming after being serialized in an Integration Response in AWS"""
```